email_alert_service/models/update_subscriber_list.rb

```diff
@@ -0,0 +1,52 @@
+require "lib/email_alert_api_helpers"
+
+class UpdateSubscriberList
+  include EmailAlertApiHelpers
+
+  def initialize(document, logger)
+    @document = document
+    @logger = logger
+  end
+
+  def trigger
+    get_subscriber_list
+    return unless subscriber_list_slug
+
+    if updateable_parameters.empty?
+      logger.info "Update cancelled for subscriber list: #{subscriber_list_slug}, no updatable paramters found"
+      return
+    end
+
+    if paramaters_need_updating?
+      logger.info "Updating subscriber list: #{subscriber_list_slug}, with: #{updateable_parameters}"
+      update_subscriber_list_details
+    else
+      logger.info "No update needed to subscriber list: #{subscriber_list_slug}"
+    end
+  end
+
+private
+
+  attr_reader :document, :logger, :subscriber_list, :subscriber_list_slug
+
+  def update_subscriber_list_details
+    Services.email_api_client.update_subscriber_list_details(
+      slug: subscriber_list_slug,
+      params: updateable_parameters,
+    )
+  rescue GdsApi::HTTPUnprocessableEntity
+    logger.info "email-alert-api returned unproessable entity updating subscriber list: #{subscriber_list_slug}, with: #{updateable_parameters}"
+  rescue GdsApi::HTTPNotFound
+    logger.info "email-alert-api cannot find subscriber list with slug #{subscriber_list_slug}"
+  end
+
+  def updateable_parameters
+    {
+      "title" => document["title"],
+    }
+  end
+
+  def paramaters_need_updating?
```

Comment:
```suggestion
  def parameters_need_updating?
```
Nitpick - `parameters` (same on line 20)

Reply:
ace, thanks!